schema/bom-1.5.schema.json

```diff
@@ -3645,7 +3645,10 @@
         "ref": {
           "title": "BOM Reference",
           "description": "References an object by its bom-ref attribute",
-          "$ref": "#/definitions/refType"
+          "anyOf": [
+            {"$ref": "#/definitions/refLinkType"},
+            {"$ref": "#/definitions/bomLinkElementType"}
```

Comment:
is any string with `iri-reference` format and a certain regex/format/pattern for BOM-Link